Simplified/NYPLMyBooksViewController.h

```diff
@@ -5,6 +5,8 @@
 - (id)initWithCoder:(NSCoder *)aDecoder NS_UNAVAILABLE;
 - (id)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil NS_UNAVAILABLE;
 
+- (void)didSelectSync;
```

Comment:
Last comment: I've reached (what I feel) is a good solution which is to also perform a book registry sync when switching libraries.